client/my-sites/subscribers/strings-for-translation.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unused-vars */
+
+// This is a temporary file that introduces strings that need to be translated.
+// It will be removed once the page is complete and the strings are translated.
+
+import { translate } from 'i18n-calypso';
+
+// username is needed for some of the strings below
+const username = 'username_placeholder';
+
+const importSubscribers = translate( 'Import subscribers' );
+const allSubscribers = translate( 'All subscribers' );
+
+const subscribers = translate( 'Subscribers' );
+const downloadCSV = translate( 'Download CSV' );
+const addSubscribers = translate( 'Add Subscribers' );
+
+const search = translate( 'Search by name, username or email…' );
+const bulkActions = translate( 'Bulk actions' );
+
+const name = translate( 'Name' );
+const status = translate( 'Status' );
+const active = translate( 'Active' );
+const inactive = translate( 'Inactive' );
+const openRate = translate( 'Open rate' );
+const subscriptionType = translate( 'Subscription type' );
+const free = translate( 'Free' );
+const monthly = translate( 'Monthly' );
+const yearly = translate( 'Yearly' );
+const all = translate( 'All' );
+const subscriptionDate = translate( 'Subscription date' );
+const since = translate( 'Since' );
+const view = translate( 'View' );
+const passwordReset = translate( 'Send password reset' );
```

Comment:
I left this string there on purpose as I felt the conversation on that topic wasn't resolved completely. But thinking about it further, I will remove the string (since it is unlikely we will be adding the password reset feature in the near future - even if we decide to go that route at the end).

Reply:
I have removed the string in https://github.com/Automattic/wp-calypso/pull/77807/commits/07d911410188860521a1b49d3458aaedb6167839.